build_scripts/build_runner.py

```diff
@@ -989,7 +989,7 @@ def main():
     log = logging.getLogger('build_runner.main')
 
     # remove duplicated values
-    target_arch = [*set(parsed_args.target_arch)]
+    target_arch = list(set(parsed_args.target_arch))
```

Comment:
Where do you import TargetArch for lines 980,981?

Reply:
Thanks, done